### ISSUE TITLE
Global variables seems to only work with data attribute not file for mocks

### ISSUE THREAD

**Issue by stefangoor** [mdasberg/ng-apimock#58]
Hi There,

Is is possible to use global variables in mock data when it comes from a file rather the data attribute?

In my scenario if I put the %%variable%% reference in a JSON object within the data: attribute it gets substituted correctly but if I have the same JSON object in a file that is loaded via the file: attribute it does not get substituted.

Is there a way to achieve this?

Many Thanks

**Comment by jeremymwells** [comment]
No, they don't seem to work that way. @mdasberg - any plans for this? PR's welcome?
